### PR TITLE
[travis] use clang-10

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,10 +41,10 @@ jobs:
 
       addons:
         apt:
-          packages: [clang-format-9]
+          packages: [clang-format-10]
```
Comment (ricab) [inline .travis.yml:44]: How about we make the dependency explicit?

```suggestion
          packages: [clang-10, clang-format-10]
```

Comment (Saviq) [reply .travis.yml:44]: No, this bit only does the formatting, it doesn't need the compiler.

**Comment by Saviq** [comment]
> Thank you, this is quite welcome, but the build is bound to fail... We can fix in separate PRs though. One such fix is in #1626 already.

Sure, I actually wanted to ensure it does, indeed, fail.

File: .travis.yml (hunk 1 above)
Comment (ricab) [reply .travis.yml:44]: Ah sorry, clang-10 is installed in the other one. I suppose this is just for conformity.

**Comment by Saviq** [comment]
Green!

bors merge a=ricab a=Saviq r=Saviq

**Pull request merged**